auth-docker-postgres-jwt/go.mod

```diff
@@ -0,0 +1,41 @@
+module app
```

Comment:
Add an entry in .github/dependabot.yml to track this file

Reply:
Sure, working on it today

Reply:
Done 👍